app/src/main/java/org/piramalswasthya/cho/adapter/PatientItemAdapter.kt

```
@@ -76,68 +46,138 @@ class PatientItemAdapter(
+                    binding.ivPatientIcon.setImageResource(R.drawable.ic_new_born_baby)
+                } else if (type == "infant") {
+                    binding.ivPatientIcon.setImageResource(R.drawable.ic_infant)
+                } else if (type == "child") {
+                    //male female check
+                    if (gender == "Male") {
+                        binding.ivPatientIcon.setImageResource(R.drawable.ic_boy)
+                    } else if (gender == "Female") {
+                        binding.ivPatientIcon.setImageResource(R.drawable.ic_girl)
+                    } else {
+
+                    }
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyElseBlock)